1458TeamCode/src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -0,0 +1,127 @@
+package frc.robot.subsystems;
+
+
+import frc.robot.Constants;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.kinematics.SwerveDriveOdometry;
+import edu.wpi.first.math.kinematics.SwerveModulePosition;
+
+import com.ctre.phoenix6.configs.Pigeon2Configuration;
+import com.ctre.phoenix6.hardware.Pigeon2;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Drive extends SubsystemBase {
+    public SwerveDriveOdometry swerveOdometry;
+    public Module[] mSwerveMods;
+    public Pigeon2 gyro;
+
+    public Drive() {
+        gyro = new Pigeon2(Constants.Swerve.pigeonID);
+        gyro.getConfigurator().apply(new Pigeon2Configuration());
+        gyro.setYaw(0);
+
+        mSwerveMods = new Module[] {
+            new Module(0, Constants.Swerve.Mod0.constants),
```

Comment:
change module number by enum

Reply:
Not sure exactly what needs to be changed, the latest push doesn't change anything relating to this